### PR TITLE
Standard community call 2 april blogpost

### PR DESCRIPTION
**Pull request by Ainali**
Scheduled for going up on Friday (but can be moved to whenever it fits).

-----
[View rendered _posts/2020-04-17-notes-from-community-call-2-april-2020.md](https://github.com/publiccodenet/blog/blob/april-2-standard-community-call/_posts/2020-04-17-notes-from-community-call-2-april-2020.md)

### CODE REVIEW

File: _posts/2020-04-10-notes-from-community-call-2-april-2020.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# 2 April 2020 community call
+
+## Attendees
+
+* [Jan](https://publiccode.net/team/jan-ainali.html) (chair)
+* [Alba](https://publiccode.net/team/alba-roza.html) (notes)
+* [Laura](https://publiccode.net/team/laura-scheske.html)
+* [Boris](https://publiccode.net/team/boris-van-hoytema.html)
+* [Eric](https://publiccode.net/team/eric-herman.html)
```
Comment (bvhme) [inline _posts/2020-04-10-notes-from-community-call-2-april-2020.md:19]: I think we should also do last names.

```suggestion
* [Jan Ainali](https://publiccode.net/team/jan-ainali.html) (chair)
* [Alba Roza](https://publiccode.net/team/alba-roza.html) (notes)
* [Laura Scheske](https://publiccode.net/team/laura-scheske.html)
* [Boris van Hoytema](https://publiccode.net/team/boris-van-hoytema.html)
* [Eric Herman](https://publiccode.net/team/eric-herman.html)
```

File: _posts/2020-04-10-notes-from-community-call-2-april-2020.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Notes from community call - 2 April 2020"
```
Comment (bvhme) [inline _posts/2020-04-10-notes-from-community-call-2-april-2020.md:2]: This should be 'Standard community call'

File: _posts/2020-04-10-notes-from-community-call-2-april-2020.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* [Jan](https://publiccode.net/team/jan-ainali.html) (chair)
+* [Alba](https://publiccode.net/team/alba-roza.html) (notes)
+* [Laura](https://publiccode.net/team/laura-scheske.html)
+* [Boris](https://publiccode.net/team/boris-van-hoytema.html)
+* [Eric](https://publiccode.net/team/eric-herman.html)
+
+## Update from the Foundation
+
+* We have launched a website for Signalen at https://signalen.org
+* [Laura](https://publiccode.net/team/laura-scheske.html) has joined the Foundation for Public Code as Membership developer
```
Comment (bvhme) [inline _posts/2020-04-10-notes-from-community-call-2-april-2020.md:24]: Both of these are not about the standard, I think they are kind of lost here. But we did talk about them so they can be here.

Comment (Ainali) [reply _posts/2020-04-10-notes-from-community-call-2-april-2020.md:24]: They are not, but they are inline with the upcoming changes that you have approved. See [the section Meeting preparations](https://github.com/publiccodenet/about/pull/583/files#diff-9b62c1e44831737ce8ace270c91e17aa). Should those be changed?

File: _posts/2020-04-10-notes-from-community-call-2-april-2020.md (hunk 2 above)
Comment (Ainali) [reply _posts/2020-04-10-notes-from-community-call-2-april-2020.md:2]: The title is harmonized with previous community calls. If this change is made, should I also change previous blog posts?

Comment (bvhme) [reply _posts/2020-04-10-notes-from-community-call-2-april-2020.md:2]: I think for now this is fine, I didn't realise we haven't merged the 'splitting the community calls' yet.

File: _posts/2020-04-10-notes-from-community-call-2-april-2020.md (hunk 3 above)
Comment (bvhme) [reply _posts/2020-04-10-notes-from-community-call-2-april-2020.md:24]: Same.

File: _posts/2020-04-10-notes-from-community-call-2-april-2020.md (hunk 2 above)
Comment (bvhme) [reply _posts/2020-04-10-notes-from-community-call-2-april-2020.md:2]: Same here

**Comment by ElenaFdR** [comment]
Shiny!

**Pull request merged**